### PR TITLE
Fix Laravel Mix code samples

### PR DESCRIPTION
**Pull request by pawelmysior**
1. Changed the Less code sample source and output paths and also added a missing semicolon.
2. Changed the Sass code sample Tailwind config path and fixed indentation.

### CODE REVIEW

File: docs/source/docs/installation.blade.md [hunk 1]
```diff
@@ -157,12 +157,12 @@ If you're using a preprocessor, use the `options` method to add `tailwindcss` as
 ```js
 var tailwindcss = require('tailwindcss');
 
-mix.less('source/_assets/less/main.less', 'source/css')
+mix.less('resources/assets/less/app.less', 'public/css')
```
Comment (mkarnicki) [inline docs/source/docs/installation.blade.md]: I would go further and even change app.less to app.css, as you can use plain css with Tailwind.

**Review by mkarnicki — APPROVED** [review]
Yeah, noticed these differences too when I used the example. Very nice.

File: docs/source/docs/installation.blade.md (hunk 1 above)
Comment (PawelMysior) [reply docs/source/docs/installation.blade.md]: While it's true that you might not need a preprocessor with the way that Tailwind works, this example is specific to using Less. And there is already a plain css example above this one.

Comment (mkarnicki) [reply docs/source/docs/installation.blade.md]: Ach, gotcha!

**Review by devcircus — COMMENTED** [review]
The config can be stored anywhere. ```path/to/config.js``` is more accurate.

**Comment by mkarnicki** [comment]
@devcircus But that change makes it consistent with the rest of the documentation.

**Comment by adamwathan** [comment]
Awesome, thanks!

**Pull request merged**